UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/FunctionalTests/CallEssEndPoint.cs

```diff
@@ -37,6 +39,9 @@ public async Task Setup()
             EssJwtToken = await authTokenProvider.GetEssToken();
             FssJwtToken = await authTokenProvider.GetFssToken();
 
+            HttpResponseMessage apiResponse = MockHelper.ConfigureFM(posWebJob.MockApiBaseUrl, posWebJob.FMConfigurationValidProductIdentifier);
+            apiResponse.StatusCode.Should().Be((HttpStatusCode)200);
+
             unpResponse = await getunp.GetJwtAuthUnpToken(fleet.baseUrl, userCredentialsBytes, fleet.subscriptionKey);
             string unpToken = await unpResponse.DeserializeAsyncToken();
             HttpResponseMessage httpResponse = await getcat.GetCatalogueEndpoint(fleet.baseUrl, unpToken, fleet.subscriptionKey);
```

Comment:
at line 50 there is a call to the runwebjob. this abstracts the task.delay of 70 sec.

Reply:
70 seconds of delay is added to wait so that webjob gets executed before we run the FTs. If we dont add this delay, FTs would get executed before webjob complets and creates batches in stub and that would fail the assertions wrtitten in the FTs.

Generally, webjob takes 50 seconds to get complete for 10 productidentifiers but we have added 20 seconds of buffer time.

Happy to discuss if needed.